libs/scripts/scripts.js

```diff
@@ -38,6 +38,34 @@ export async function loadLCP(blocks) {
   }
 }
 
+export async function fetchLib(type) {
+  const namespace = `${type}Library`;
+  if (!type || window[namespace]) return;
+  const resp = await fetch(`${window.location.origin}/docs/${type}-library.json`);
+  if (resp.ok) {
+    try {
+      const json = await resp.json();
+      window[namespace] = {};
```

Comment:
I think we should put any globals under `window.milo` to keep it all in one place
```suggestion
      window.milo = window.milo || {};
      window.milo[namespace] = {};
```